Source/dictionary_service.json

```diff
@@ -0,0 +1,60 @@
+{
+  "services": [
+    {
+      "name": "Dictionary app",
+      "url_template": "dict://(encoded)"
+    },
+    {
+      "name": "MOE Dict",
```

Comment:
I was wondering if this should be called `"moedict.tw"` since "MOE Dict" sounds like the officical MOE dictionary. Or should this simply be called "萌典" without localization just like "教育部重編國語詞典修訂本" below?

Calling the three entries 萌典、萌典（台語）、萌典（客語） also removes the need to localize them in English.